core/common/src/main/java/alluxio/util/ConfigurationUtils.java

```diff
@@ -41,25 +42,27 @@
   private ConfigurationUtils() {} // prevent instantiation
 
   /**
-   * Loads properties from resource. This method will search Classpath for the properties file with
-   * the given resourceName.
+   * Loads properties from a resource.
    *
-   * @param resourceName filename of the properties file
+   * @param resource url of the properties file
    * @return a set of properties on success, or null if failed
    */
-  public static Properties loadPropertiesFromResource(String resourceName) {
-    Properties properties = new Properties();
-
-    InputStream inputStream =
-        Configuration.class.getClassLoader().getResourceAsStream(resourceName);
+  public static Properties loadPropertiesFromResource(URL resource) {
```

Comment:
`@Nullable`

Reply:
possible to consolidate this one with `loadPropertiesFromFile` so we maintain less code?
e.g., consolidate into one method takes an input stream ?

Reply:
refactored and added `@Nullable`